kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/CaseServicesClient.java

```diff
@@ -0,0 +1,117 @@
+
+    void removeCaseInstanceData(String containerId, String caseId, String... names);
+
+    void addDynamicUserTask(String containerId, String caseId, String name, String description, String actors, String groups, Map<String, Object> data);
+
+    void addDynamicTask(String containerId, String caseId, String nodeType, String name, Map<String, Object> data);
+
+    void addDynamicUserTaskToStage(String containerId, String caseId, String stageId, String name, String description, String actors, String groups, Map<String, Object> data);
+
+    void addDynamicTaskToStage(String containerId, String caseId, String stageId, String nodeType, String name, Map<String, Object> data);
+
+    void addDynamicSubProcess(String containerId, String caseId, String processId, Map<String, Object> data);
+
+    void addDynamicSubProcessToStage(String containerId, String caseId, String stageId,  String processId, Map<String, Object> data);
+
+    void triggerAdHocFragment(String containerId, String caseId, String adHocName, Map<String, Object> data);
+
+    void triggerAdHocFragmentInStage(String containerId, String caseId, String stageId, String adHocName, Map<String, Object> data);
+
+    List<CaseMilestone> getMilestones(String containerId, String caseId, boolean achievedOnly, Integer page, Integer pageSize);
+
+    List<CaseStage> getStages(String containerId, String caseId, boolean activeOnly, Integer page, Integer pageSize);
+
+    List<CaseAdHocFragment> getAdHocFragments(String containerId, String caseId);
+
+    List<CaseRoleAssignment> getRoleAssignments(String containerId, String caseId);
+
+    List<NodeInstance> getActiveNodes(String containerId, String caseId, Integer page, Integer pageSize);
+
+    List<ProcessInstance> getActiveProcessInstances(String containerId, String caseId, Integer page, Integer pageSize);
+
+    List<ProcessInstance> getProcessInstances(String containerId, String caseId, List<Integer> status, Integer page, Integer pageSize);
+
+    void assignUserToRole(String containerId, String caseId, String roleName, String user);
+
+    void assignGroupToRole(String containerId, String caseId, String roleName, String group);
+
+    void removeUserFromRole(String containerId, String caseId, String roleName, String user);
+
+    void removeGroupFromRole(String containerId, String caseId, String roleName, String group);
+
+    List<CaseComment> getComments(String containerId, String caseId, Integer page, Integer pageSize);
+
+    void addComment(String containerId, String caseId, String author, String text);
+
+    void removeComment(String containerId, String caseId, String commentId);
+
+    List<CaseInstance> getCaseInstances(Integer page, Integer pageSize);
+
+    List<CaseInstance> getCaseInstances(List<Integer> status, Integer page, Integer pageSize);
+
+    List<CaseInstance> getCaseInstancesOwnedBy(String owner, List<Integer> status, Integer page, Integer pageSize);
```

Comment:
the one above is pretty much getCaseInstancesByStatus it just does not have the ByStatus in method name